I chose to analyze the phylogeny of Clermontia, a Hawaiian lobeliad plant group. The dataset is located in the shared research drive named givnish file. First, I duplicated the lobeliads folder to make sure I will not damage or change any raw data. The new duplicated folder is renamed as “Bing_Lobeliads_copy”. This project will work on this folder. This folder contains a excel file, named “Hawaiian lobelias samples 2021”, lists the sample ID and species name, because this dataset is large. The folder called “assemblies” contain assembled sequences, and the folder named “raw reads” contains all the raw reads data. 

For this final project, I will directly use the assembled data generated by Nisa and cite the paper. The data have been cleaned and assembled. For the purpose of learning, I will run the raw data through FastQC first. In this case, I chose to download the unzipped file from the “raw read” folder and zip the fastq file, and then input the file into FastQC. The output reports are uploaded into github under files names “FastQC_results”. 

I will select for species of Clermontia and a couple of outgroups including, Cyanea pills (I12677) and Cyanea leptostegia (I19593) , to do this project. These two species have been used as outgroups in previous publications. I first manually moved all the files of Clermontia and outgroups into a folder within folder “RawReads”, called “Clermontia”.

First, I tried to unzip *.fastq.gz in different folders with terminal with: 
$ for i in *.fastq.gz; do gunzip "$i" -d "${i%%.zip}"
This line does not work 

Then, 
$ for i in *.gz; do gunzip "$i" -d "${i%%.zip}” — Does not work either

I decided to manually unzip some of the files to get fastq file for the purpose of learning. I achieved ten folder — Now I have two Fastq files within each folder. 

I tried to run multiple files on different branches simultaneously though shell but failed, and then I decided to run it via the user interplay. It took 4 hours to run 10 samples, and there might be a easier way. 






 

 



